frontend/src/app/patients/EditPatient.test.tsx

```diff
@@ -682,6 +682,58 @@ describe("EditPatient", () => {
       expect(await screen.findByText("Conduct tests")).toBeInTheDocument();
       expect(screen.queryByText(PATIENT_TERM_CAP)).not.toBeInTheDocument();
       expect(screen.queryByText("Save and start test")).not.toBeInTheDocument();
+      expect(screen.queryByText("Start test")).not.toBeInTheDocument();
+    });
+  });
+
+  describe("Start test and Save and start test from edit patient", () => {
+    const renderWithUser = (
+      elementToRender:
```

Comment:
I think we can get around having to do this complicated type definition by just moving the `const elementToRender` declaration into the describe block and then calling it directly, rather than passing it in as a parameter since it doesn't look like we're varying which element we're rendering between tests. See line 422 for an example

Reply:
Fixed -- thanks for catching that this makes things way cleaner 🧹